packages/compartment-mapper/src/parse-cjs-shared-export-wrapper.js

```diff
@@ -8,6 +8,47 @@ const { freeze, keys, create, hasOwnProperty } = Object;
  * @returns {boolean}
  */
 const has = (object, key) => apply(hasOwnProperty, object, [key]);
+const removeProtocol = url => {
+  if (url.substring(0, 7) === 'file://') {
```

Comment:
`indexOf('file://') !== -1` has the virtue of not allocating, but I don’t feel strongly.

Reply:
appreciated